adding-autodeployment/README.md

```diff
@@ -0,0 +1,100 @@
+# Autodeployment Guide
+## What to do?
+1. [Get deployer credentials](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#1-getting-deployer-credentials)
+2. [Update `.travis.yml`](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#2-update-travisyml)
+3. [Add `deploy.sh`](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#3-add-deploysh)
+4. [Add manifests](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#4-add-manifests)
+
+## What is this  guide for?
+This guide is to help developers implement [Contraband's Autopilot](https://github.com/contraband/autopilot) plugin on a [cloud.gov](https://cloud.gov/) hosted app. In the future we hope to provide you with a small script to generate this all for you!
+
+## Pre-reqs
+This guide assumes that you already have:
+- a git repository
+- continuous integration services on [Travis CI](https://travis-ci.org/).
+- a cloud.gov account.
+
+## 1. Getting deployer credentials
+Use the [instructions on Cloud.gov](https://cloud.gov/docs/apps/continuous-deployment/#provisioning-deployment-credentials) to create a deployer account for your app.
+
+In general, follow the rules of configuration as code. But because of `travis encrypt` escape issues, we suggest that you store these credentials in the travis environement variable gui found here: https://travis-ci.org/18F/YOUR_REPO/settings. Make sure that the *Display value in build log* setting is **OFF** for this value.
```

Comment:
It's not the most helpful, but it may be worth linking to https://docs.travis-ci.com/user/encryption-keys/#Note-on-escaping-certain-symbols here. Hopefully, [this pull request](https://github.com/18F/deployer-account-broker/pull/15) will resolve the escaping risk soon, though.